static/riotjs/wizard/create_team.tag

```diff
@@ -104,13 +94,35 @@
                     window.location='/klasses/wizard/' + KLASS + '/enroll'
                 })
                 .fail(function (response) {
-                    console.log(response)
                     Object.keys(response.responseJSON).forEach(function (key) {
                         toastr.error("Error with " + key + "! " + response.responseJSON[key])
                     });
                 })
         }
 
+        self.get_eligible_leaders = () => {
+            return self.klass.enrolled_students.filter( (student) => {
+                return student.on_team
+            })
+                .map( (student) => {
+                    return {
+                        name: _.get(student, 'user.first_name') + ' ' + _.get(student, 'user.last_name'),
+                        value: student.id,
+                        selected: student.leader
+                    }
+                })
+        }
+
+        self.leader_dropdown_on_change = (value, text, $selectedItem) => {
+            let leader_index = self.klass.enrolled_students.findIndex( (student) => {
+                return student.id == value
+            })
+
+            for (let i = 0; i < self.klass.enrolled_students.length; i++) {
```

Comment:
Mark the new student as the leader on the `enrolled_students` array.

Reply:
👍